Tests/ParameterSpaces/knot_vector_mock.cpp

```diff
@@ -473,9 +474,9 @@ void AKnotVectorMock::NurbsBookExe3_8_0Subdivided() {
   EXPECT_CALL(*this, GetUniqueKnotsMock(IsGe0_0AndLt0_5)).WillRepeatedly(Return(kUniqueKnots3_8));
 
   EXPECT_CALL(*this, RemoveMock(k0_5, kMultiplicity_, IsGe0_0AndLt0_5)).Times(AtMost(1))
-      .WillOnce(InvokeWithoutArgs(this, &AKnotVectorMock::NurbsBookExe3_8_0));
+      .WillOnce(DoAll(InvokeWithoutArgs(this, &AKnotVectorMock::NurbsBookExe3_8_0), Return(kMultiplicity_)));
   EXPECT_CALL(*this, RemoveMock(k0_5, kMultiplicity2, IsGe0_0AndLt0_5)).Times(AtMost(1))
-      .WillOnce(InvokeWithoutArgs(this, &AKnotVectorMock::NurbsBookExa2_1));
+      .WillOnce(DoAll(InvokeWithoutArgs(this, &AKnotVectorMock::NurbsBookExa2_1), Return(kMultiplicity_)));
```

Comment:
This caused the test failure; we need to return `kMultiplicity2`. I fixed it and will merge this branch after pushing the fix.